README.md

```diff
@@ -1,41 +1,46 @@
-
-## Description
-
-For this assignment you will be building a bitmap (`.bmp`) reader and transformer. It will read a bitmap in from disk, run one or more color transforms on the bitmap and then write it out to a new file. This project will require the use of node buffers in order to manipulate binary data. Your project should include tests, as well as a `gulpfile.js`, `package.json`, `.eslintrc`, `README.md`, and a `.gitignore`. Make sure to run all your code through eslint. The process will look something like this:
-
-1. open the original bitmap file using fs and read it into a buffer
-2. convert the buffer header data into a Javascript Object (using constructors)
-3. run a transform on the buffer directly
-4. write the buffer to a new file
-
-The wikipedia article found here [Bitmap Specification](https://en.wikipedia.org/wiki/BMP_file_format) describes the byte specification of a "windows bitmap file." We'll be working with the simplest version, meaning no compression.
-
-* your project should have three ***(or more)*** transforms
-* invert the colors (***hint:*** subtract every color value from the max color value which is 255),
-* grayscale the colors (***hint:*** multiply each color value by a constant, just make sure your values don't go over 255)
-* (red|green|blue)scale the colors (***hint:*** same as above but only multiply one of the colors)
-
-## Bonus:
-
-* ability to handle various sized bitmap
-* ability to handle LE and BE computers with a single if statement
-* utilizes a command line interface (CLI)
-* CLI can select the transforms
+# BITMAP TRANSFORMER
+Built by TEAM MEGATRON - Khalid Mohamud, Dana Kulp, Regan O'Neill
+
+This program utilizes node.js to read a bitmap data stream, manipulate the stream, and create a new bitmap image to the filesystem.
+
+# Features Include:
+
+  - Read bitmap data stream from operating system
+  - copy data as Buffer object and transform into different data structures in order to transform image.
+  - use object constructor to allow metadata to persist and be reused within application
+  - utilize three methods to transform the original file:
+        1) INVERT COLORS
+        [Imgur](http://i.imgur.com/GPZ8S8J.png)
+        2) GREENSCALE IMAGE
+        [Imgur](http://i.imgur.com/vQXy11e.png)
+        3) GRAYSCALE IMAGE
+        [Imgur](http://i.imgur.com/W2Tu028.png)
+
+You can also:
+  - Run individual methods directly from the CLI to create one (or more) new bitmap files.
+  - run automation testing tools with gulp
+
+### Tech
+
+* [node.js] - evented I/O for the backend
+* [Gulp] - the streaming build system
+
+### Installation
+
+Bitmap Transformer requires [Node.js](https://nodejs.org/) v6+ to run.
+
+Install the dependencies and devDependencies and start the server.
+
+```sh
+$ cd bitmap-megatron
+$ npm i
+$ node index
+[optional single commands include: 'invert', 'grayscale', 'greenscale']
+```
+License
+----
+
+MIT
+
+
+**Free Software, Hell Yeah!**
```

Comment:
@khalidM3 @reganoneill @dkulp23 `README` looks good - great focus on using different formatting constructs in markdown